packages/pds/tests/seeds/basic.ts

```diff
@@ -42,6 +84,9 @@ export default async (sc: SeedClient) => {
   await sc.vote('down', carol, sc.posts[alice][1].ref)
   await sc.vote('up', carol, sc.posts[alice][2].ref)
   await sc.vote('up', dan, sc.posts[alice][1].ref)
+  await sc.vote('up', alice, sc.posts[carol][0].ref)
+  await sc.vote('up', bob, sc.posts[carol][0].ref)
+  mq && (await mq.process())
```

Comment:
here, we want to wait till the queue is done processing before moving on

this gives us a predictable order for our snapshots